PennCourses/urls/pcp-construction.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from django.views.generic import TemplateView
+
+from PennCourses.urls.base import urlpatterns
+
+
+urlpatterns = [
+    path('', TemplateView.as_view(template_name='plan/index.html')),
```

Comment:
`plan/index.html` and `plan/build/index.html` refer to two completely different `plan` directories, but that's not obvious in the code. Can we change this one to be something like `plan_construction/index.html`?